### DISCUSSION TITLE
Dropdown

### DISCUSSION THREAD

**Discussion by abdurraheemj** [seleniumbase/SeleniumBase#1020] [Q&A] [answered]
Hi Michael,
I really have problem accessing the list which is in dropdown. i used the the selectbyvalue or selectbytext. the problem is values are in <option> tag. i used seleniumIDE to make it easier. its not giving me right solution. 
for example: 
dropdown1  has mango,apple,banana,tomato
dropdown2 has onions,ginger,tomato
i wanna select apple from dropdown1 and tomato from dropdown2
if i use seleniumIDE code. first it choose apple in dropdown1 and just change the value again in dropdown1 to tomato. 
it has the correct selectors .Thanks in Advance

**Comment by mdmintz** [comment, marked as answer]
Hi @abdurraheemj See an example of ``self.select_option_by_text(selector, text)`` in https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_demo_site.py

Also, you may run into cases where the ``:contains()`` selector would come in useful. See https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_contains_selector.py for an example of that.
